src/vaadin-button.html

```diff
@@ -176,6 +176,15 @@
         get focusElement() {
           return this.$.button;
         }
+
+        /**
+         * @protected
+         */
+        click() {
```

Comment:
I feel like this should be handled in `vaadin-control-state-mixin` repo, where the `disabled` state is managed. 
It might be that someone would try to do the same with e.g. `vaadin-checkbox` and `vaadin-radio-button`.

Reply:
👍

Reply:
https://github.com/vaadin/vaadin-control-state-mixin/pull/49